modules/network-flow-matrix.adoc

```diff
@@ -6,26 +6,100 @@
 [id="network-flow-matrix_{context}"]
 = {product-title} network flow matrix
 
-The network flow matrix describes the ingress flows to {product-title} services.
-The network information in the matrix is accurate for both bare-metal and cloud environments.
-Use the information in the network flow matrix to help you manage ingress traffic.
-You can restrict ingress traffic to essential flows to improve network security.
+The following network flow matrixes describe the ingress flows to {product-title} services for the following environments:
 
-To view or download the raw CSV content, see link:https://raw.githubusercontent.com/openshift/openshift-docs/main/snippets/network-flow-matrix.csv[this resource].
+* {product-title} on bare metal 
+* {sno-caps} on bare metal
+* {product-title} on {aws-first}
+* {sno-caps} on {aws-short}
 
-Additionally, consider the following dynamic port ranges when managing ingress traffic:
+Use the information in the appropriate network flow matrix to help you manage ingress traffic for your specific environment. You can restrict ingress traffic to essential flows to improve network security.
+
+Additionally, consider the following dynamic port ranges when managing ingress traffic for both bare metal and cloud environments:
 
 * `9000-9999`: Host level services
 * `30000-32767`: Kubernetes node ports
 * `49152-65535`: Dynamic or private ports
 
+To view or download the complete raw CSV content for an environment, see the following resources:
+
+* link:https://raw.githubusercontent.com/openshift-kni/commatrix/release-4.18/docs/stable/raw/bm.csv[{product-title} on bare metal]
+
+* link:https://raw.githubusercontent.com/openshift-kni/commatrix/release-4.18/docs/stable/raw/bm-sno.csv[{sno-caps} on bare metal]
+
+* link:https://raw.githubusercontent.com/openshift-kni/commatrix/release-4.18/docs/stable/raw/aws.csv[{product-title} on {aws-short}]
+
+* link:https://raw.githubusercontent.com/openshift-kni/commatrix/release-4.18/docs/stable/raw/aws-sno.csv[{sno-caps} on {aws-short}]
+
+[NOTE]
+====
+The network flow matrixes describe ingress traffic flows for a base {product-title} or {sno} installation. It does not describe network flows for additional components, such as optional Operators available from the Red Hat Marketplace. The matrixes do not apply for Hosted-Control-Plane, MicroShift, or standalone clusters.
```

Comment:
```suggestion
The network flow matrixes describe ingress traffic flows for a base {product-title} or {sno} installation. It does not describe network flows for additional components, such as optional Operators available from the Red Hat Marketplace. The matrixes do not apply for {hcp}, MicroShift, or standalone clusters.
```

Hosted control planes is written as -- 
- **h**osted control planes: In subsequent occurrences ;  attribute {hcp}
- **H**osted control planes: Beginning of sentence, title, and heading ; {hcp-capital}